routemaster/client/connection.rb

```diff
@@ -43,7 +43,7 @@ def subscribe(options)
           end
 
           unless response.success?
-            raise 'subscribe rejected'
+            raise "subscribe rejected (#{response.status})"
```

Comment:
Good idea. I was mainly replicating the existing pattern but totally happy to do that.